scalecube-organization-api/src/main/java/io/scalecube/account/api/Role.java

```diff
@@ -1,17 +1,17 @@
 package io.scalecube.account.api;
 
 public enum Role {
-  Owner("owner"),
-  Admin("admin"),
-  Member("member");
+  Owner(300),

```

Comment:
not sure i like this ranking technique
maybe role and rank are different thing?

Reply:
@ronenhamias we have some vertical priority structure of these roles, so in the beginning this solution with ranks pretty well. It's better than just rely on the enum ordinal.
But ok maybe let's change this in another PR after some discussion and a concrete proposal.

Reply:
well ok i am just afraid this should be solved not as part of this lib

Reply:
flat role list is assumed in some systems roles are arranged in hierarchies